packages/host/app/services/matrix-service.ts

```diff
@@ -393,6 +400,31 @@ export default class MatrixService extends Service {
     }
   }
 
+  async sendCommandResultMessage(roomId: string, eventId: string, result: any) {
```

Comment:
i know the `result` is pretty free form, but it is still definitely JSON. let's make sure the type reflects that so that if `undefineds`, `string` or other obviously wrong stuff slips in there, we'll see it. Perhaps, `Record<string, any>` would suffice.

Reply:
We could use the same def we do on the other side, 

https://github.com/cardstack/boxel/blob/1f42028f3804f8c3a60413970e064014fe58b835/packages/base/command.gts#L6

which specifies the types to be ~compatible with json, so no functions etc.